tests/bluetooth/bsim_bt/edtt_ble_test_app/tests_scripts/_controller_tests_inner.sh

```diff
@@ -4,16 +4,32 @@
 
 # Common part of the test scripts for some of the EDTT tests
 # in which 2 controller only builds of the stack are run against each other
-VERBOSITY_LEVEL=2
+VERBOSITY_LEVEL=${VERBOSITY_LEVEL:-2}
+VERBOSITY_LEVEL_EDTT=${VERBOSITY_LEVEL_EDTT:-${VERBOSITY_LEVEL}}
+VERBOSITY_LEVEL_BRIDGE=${VERBOSITY_LEVEL_BRIDGE:-${VERBOSITY_LEVEL}}
+VERBOSITY_LEVEL_PHY=${VERBOSITY_LEVEL_PHY:-${VERBOSITY_LEVEL}}
+VERBOSITY_LEVEL_DEVS=${VERBOSITY_LEVEL_DEVS:-${VERBOSITY_LEVEL}}
+VERBOSITY_LEVEL_DEV1=${VERBOSITY_LEVEL_1:-${VERBOSITY_LEVEL_DEVS}}
+VERBOSITY_LEVEL_DEV2=${VERBOSITY_LEVEL_2:-${VERBOSITY_LEVEL_DEVS}}
+
 PROCESS_IDS=""; EXIT_CODE=0
 
 function Execute(){
+  local rr=
+  if [ "rr" = "$1" ]; then
+    local devno=$2
+    shift 2
+    local exe=$(basename $1)
+    local out=/tmp/rr/$$-${SIMULATION_ID}-${exe}-d_${devno}
+    rm -rf ${out}
+    rr="rr record -o ${out}"
```

Comment:
if we are paranoid we could add a check to see if rr is installed

Reply:
ACK